bcp47-orphans/library/Data/BCP47/PathPieces.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Data.BCP47.PathPieces () where
+
+import Data.BCP47 (BCP47, fromText, toText)
+import Web.PathPieces (PathPiece(..))
+
+instance PathPiece BCP47 where
+  fromPathPiece = either (const Nothing) Just . fromText
```

Comment:
I have a super low bar for bringing in `Control.Error.Util.hush`. Just saying ;)